CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-cryptoauthlib INTERFACE)
+
+add_subdirectory(cryptoauthlib)
```

Comment:
I had to remember that this directory gets added after `mbed deploy` (or the equivalent command in mbed-tools). Is it worth adding a comment?